lib/id3c/cli/command/identifier.py

```diff
@@ -70,6 +76,9 @@ def mint(set_name, count, *, labels, quiet, dry_run):
     An alternative instance may be used by setting the LABEL_API environment
     variable to the instance URL.
 
+    If --small is requested, the printable barcode labels will use the "small"
+    version of the layout, if available.
```

Comment:
To be more upfront and clear with the user, I would be inclined to have it throw a `notimplemented` if this option is requested where its not supported yet rather than silently use the normal label.

Reply:
This is now implemented.